MANIFEST.in

```diff
@@ -3,3 +3,4 @@ include man/*
 include LICENSE
 include README.md
 include config/convert2rhel.ini
+exclude pyproject.toml
```

Comment:
Well, this file mainly contains overrides for the tools that we use during development. I thought that this was not necessary in the tarball, as it is not really being used by anything, specially because we haven't migrated to use only pyproject.toml with setuptools and drop the `setup.{cfg,py}` ones.

Reply:
does that make sense to you, @abadger?

Reply:
I think I would include pyproject.toml if the only issue is whether it is necessary.  On a practical side, I know that packagers like to run the tests that upstreams have produced so leaving out some of the configuration for tools that we use in our pre-commit configuration doesn't seem right (Although packagers *probably* will only run pytest and not something like `pre-commit run --all-files`).

I also know that the GPL requires corresponding source which includes all the "build files" required to transform the source.  With configuration of developer tools for an interpreted language, the line is not as clear as with compiled code but I like to err on the side of caution there.